include/blmc_robots/common_programs_header.hpp

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
if you change the filename, it would be good to do it in a separate commit. if you change the filename and change the file content at the same time it makes reviewing very hard.